.github/workflows/unittest.yml

```diff
@@ -3,7 +3,7 @@ on:
   push:
     branches:
     - master
-  pull_request:
+  pull_request_target:
```

Comment:
You shouldn't change `pull_request` to `pull_request_target`. If you use `pull_request_target`, then you are not supposed to checkout PR code for security reason. The reason the comment didn't get triggered in this PR is because the build doc process failed for some reason

Reply:
+1 on no change to `pull_request`. For the failure, the first time it fails on comment on PR due to AWS permission error (that build is not long), but the second time fails on building doc (takes several hours). @josephevans Please change here back and trigger another run, let's investigate the reason.

Reply:
Just checked the log from aws batch. The job succeeded but took a long time(3 hrs). Should be able to merge once the workflow change is reverted

Reply:
Cool, thanks for your help, @yinweisu , appreciate it.

Reply:
Thanks for the insights, I've reverted this change.